lib/redacted_articles.json

```diff
@@ -0,0 +1,1067 @@
+[
```

Comment:
We might be able to put this in the deployment-charts repo in the eventstreams helmfile.  Will discuss on ticket.